app/views/case_contacts/form/_contact_types.html.erb

```diff
@@ -1,5 +1,5 @@
 <div id="contact-type-form">
-  <h2 id="contact-type-label"><label>2. Select All Contact Types</label><span class="red-letter"> *</span></h2>
+  <h4 id="contact-type-label"><label>2. Select All Contact Types</label><span class="red-letter"> *</span></h4>
```

Comment:
Is this `h4` semantically correct, or was it changed for font-size reasons?

Reply:
The redesign asks for these headers to be h4. https://docs.google.com/document/d/1gP9eLd_wzVBKfyNe5pwPJPjezcHMxNBugD6GqxAX3n4/edit#heading=h.5xzbwuobd5ze